### PR TITLE
Add a simple run instruction to README.md

### PR DESCRIPTION
**Pull request by kenoir**
## Who is this for?

Developers unfamiliar with developing on `wellcomecollection.org`

## What is it doing for them?

Providing simple instructions for how to get started

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,20 @@ discussions, and more.
...
 - Wellcome Collection's design system. [`code`](./cardigan).
 
+## Developing
+
+This project uses [Yarn](https://yarnpkg.com/lang/en/). You'll need to install it before proceeding.
+
+To start a hot reload of a service run:
+
+```
+# Install dependencies
```
Comment (jamesgorrie) [inline README.md]: As we use workspaces, dependencies are managed from the root directory.
I also haven't seen the run command anywhere?

Suggestion
---

## Local development

We use [Yarn](https://yarnpkg.com/lang/en/) to manage out external dependencies.

We then use [Yarn Workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) to manage our [local, common dependencies](https://github.com/wellcometrust/wellcomecollection.org/tree/master/common).

To run a project, from the root directory:
```bash
yarn install
# yarn run {appName = catalogue|content}
# e.g.
yarn run catalogue
```

---


I might add a piece about how the webapps work with middleware etc.

Comment (kenoir) [reply README.md]: Much nicer - changed as per suggestion. I think you can offer code changes directly on PRs too.

File: README.md [hunk 2]
```diff
@@ -33,6 +33,20 @@ discussions, and more.
...
+## Local development
+
+We use [Yarn](https://yarnpkg.com/lang/en/) to manage out external dependencies.
+
+We then use [Yarn Workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) to manage our [local, common dependencies](https://github.com/wellcometrust/wellcomecollection.org/tree/master/common).
+
+To run a project, from the root directory:
+```bash
+yarn install
+# yarn run {appName = catalogue|content}
```
Comment (davidpmccormick) [inline README.md]: don't need the `run`

File: README.md [hunk 3]
```diff
@@ -33,6 +33,20 @@ discussions, and more.
...
+# e.g.
+yarn run catalogue
+```
+
```
Comment (gestchild) [inline README.md]:
```suggestion
By default the above command will run the application at localhost:3000, you can specify a port from the application directory with the following command:
`yarn dev -p 3001`
This is useful if you want to run both the catalogue and content apps simultaneously.
```

File: README.md [hunk 4]
```diff
@@ -33,6 +33,20 @@ discussions, and more.
 ### [Cardigan](https://cardigan.wellcomecollection.org)
 - Wellcome Collection's design system. [`code`](./cardigan).
 
+## Local development
+
+We use [Yarn](https://yarnpkg.com/lang/en/) to manage out external dependencies.
```
Comment (jamesgorrie) [inline README.md]:
```suggestion
We use [Yarn](https://yarnpkg.com/lang/en/) to manage our external dependencies.
```

File: README.md (hunk 2 above)
Comment (jamesgorrie) [reply README.md]:
```suggestion
# yarn {appName = catalogue|content}
```

**Pull request merged**